projects/observability/src/shared/components/explorer-filter-on-hover/explorer-filter-on-hover.component.ts

```diff
@@ -0,0 +1,41 @@
+import { ChangeDetectionStrategy, Component, Inject } from '@angular/core';
+import { IconType } from '@hypertrace/assets-library';
+import { NavigationService } from '@hypertrace/common';
+import { IconSize, POPOVER_DATA } from '@hypertrace/components';
+import { ExplorerService } from '../../../pages/explorer/explorer-service';
+import { ExplorerFilterOnHoverData } from './explorer-filter-on-hover.directive';
+
+@Component({
+  selector: 'ht-explorer-filter-on-hover',
```

Comment:
I think separating the trigger and target can make this work much more precisely, and will allow decoupling the hover behavior with a specific component like an explorer drill down icon.